### ISSUE TITLE
Uklesh doesn't spawn Khalum

### ISSUE THREAD

**Issue by gigabite1123** [project-rubika/issues#392]
Verified post server restart:
2x Uklesh the frozen spawn. Both can be killed and only 1 generates Khalum. Only 1 on respawn?

Player level exceeded 60 after kill, set to 55 which switched "playfield server"

"Locating next playfield server.
Located."

Only 1 uklesh was present. 
```
Command: getfull
/getfull [name/id]
Saece [59 Fixer]
------------------------------------a
    strength 	825
    stamina 	806
    agility 	815
    sense 	857
    psychic 	803
    intelligence 	806
    nextxp 	2954450
    lastsavexp 	203500
    xp 	2777350
    waitstate 	2
    lastxp 	2777350
    lastsaved 	2777350
    minmembers 	4
    isfightingme 	4
    charstate 	0
    exitstatel 	-1071380597
    externaldoorinstance 	-1073216889
    externalplayfieldinstance 	647
    driveground 	2209
    drivewater 	2209
    mapnavigation 	2207
    playfieldtype 	1
    selectedtargettype 	0
    expansionplayfield 	0
    treatment 	2204
    concealment 	2211
    breakingentry 	2205
    disarmtrap 	2249
    perception 	2278
    driveair 	2209
    biologicalmetamorphose 	2254
    psychologicalmodification 	2231
    materialcreation 	2201
    materiallocation 	2225
    senseimprovement 	2238
    firstaid 	2234
    nanoprogramming 	2201
    computerliteracy 	2201
    psychology 	2207
    chemistry 	2201
    tutoring 	2243
    materialmetamorphose 	2268
    runspeed 	1291
    mechanicalengineering 	2202
    electricalengineering 	2202
    fieldquantumphysics 	2201
    weaponsmithing 	2203
    pharmaceuticals 	2201
    physicalprowessinitiative 	2209
    nanoprowessinitiative 	2210
    dodge 	2314
    evade 	2340
    duck 	2273
    nanoac 	2027
    burst 	2269
    fullauto 	2204
    bowspecialattack 	2208
    lr_multipleweapon 	2270
    closecombatinitiative 	2209
    distanceweaponinitiative 	2277
    submachinegun 	2336
    shotgun 	2204
    rifle 	2207
    lr_energyweapon 	2206
    flingshot 	2230
    aimedshot 	2274
    throwingknife 	2206
    grenade 	2207
    throwngrapplingweapons 	2204
    bow 	2208
    pistol 	2207
    assaultrifle 	2205
    twohandedgedweapons 	2204
    meleeenergyweapon 	2201
    parry 	2208
    sneakattack 	2205
    meleemultiple 	2204
    fastattack 	2207
    adventuring 	2203
    swim 	2202
    onehandbluntweapons 	2204
    onehandedgedweapon 	2203
    piercing 	2203
    twohandbluntweapons 	2203
    nanoenergypool 	1002
    maxnanoenergy 	2003193
    martialarts 	2266
    brawl 	2272
    riposte 	2209
    dimach 	2211
    health 	2202256
    selectedtarget 	639
    currentmovementmode 	3
    side 	1
    life 	2103372
    bodydevelopment 	1001
    level 	59
    sex 	2
    visualsex 	2
    titlelevel 	3
    equippedweapontype 	546
    EquippedRHWeapon 	546
    expansion 	3
    fatness 	2
    breed 	2
    visualbreed 	2
    profession 	4
    visualprofession 	4
    flags 	528961
    monsterscale 	100
    currentnano 	106600
    headmesh 	40271
    cash 	33639
    gmlevel 	511
    maxncu 	2224
    monsterdata 	0
    team 	0
    healdelta 	100019
    reflectradiationac 	200
    reflectprojectileac 	200
    reflectenergyac 	200
    reflectfireac 	200
    reflectcoldac 	200
    reflectchemicalac 	200
    reflectpoisonac 	200
    reflectmeleeac 	200
    projectileac 	20000
    energyac 	20000
    meleeac 	20000
    coldac 	20000
    fireac 	20000
    chemicalac 	20508
    radiationac 	20000
    poisonac 	20000
    xpmodifier 	500
    beltslots 	6
    race 	1
    accountflags 	1234567890
    VisualFlags 	31
    clan 	0
    healinterval 	3
    nanointerval 	2
    aggdef 	100
    state 	0
    unarmedtemplateinstance 	0
    invaderskilled 	0
    killedbyinvaders 	0
    vp 	0
    unsavedxp 	0
    nanofocuslevel 	0
    specialization 	0
    shadowbreedtemplate 	0
    shadowbreed 	0
    lastperkresettime 	0
    socialstatus 	0
    playeroptions 	0
    tempsaveteamid 	0
    tempsaveplayfield 	0
    tempsavex 	0
    tempsavey 	0
    PVPDuelKills 	0
    PVPDuelDeaths 	0
    PVPProfessionDuelKills 	0
    PVPProfessionDuelDeaths 	0
    PVPRankedSoloKills 	0
    PVPRankedSoloDeaths 	0
    PVPRankedTeamKills 	0
    PVPRankedTeamDeaths 	0
    PVPSoloScore 	0
    PVPTeamScore 	0
    PVPDuelScore 	0
    unreadmailcount 	0
    lastmailchecktime 	1283065897
    savedxp 	0
    features 	6
    apartmentsallowed 	1
    apartmentsowned 	0
    lastconcreteplayfieldinstance 	0
    mapoptions 	0
    mapareapart1 	0
    mapareapart2 	0
    mapareapart3 	0
    mapareapart4 	0
    missionbits1 	0
    missionbits2 	0
    missionbits3 	0
    missionbits4 	0
    missionbits5 	0
    missionbits6 	0
    missionbits7 	0
    missionbits8 	0
    missionbits9 	0
    missionbits10 	0
    missionbits11 	0
    missionbits12 	0
    sessiontime 	1234567890
    autoattackflags 	5
    personalresearchlevel 	0
    globalresearchlevel 	0
    personalresearchgoal 	0
    globalresearchgoal 	0
    battlestationside 	0
    battlestationrep 	10
    members 	999
    veteranpoints 	0
    monthspaid 	0
    PaidPoints 	0
    xpkillrange 	5
    inplay 	0
    attitude 	0
    alignment 	0
    icon 	0
    mesh 	17530
    deadtimer 	0
    currentmass 	0
    itemtype 	0
    previoushealth 	50
    currentstate 	0
    age 	0
    attackspeed 	5
    metatype 	0
    rp 	0
    specialcondition 	1
    sk 	0
    clanredeemed 	0
    clanconserver 	0
    clandevoted 	0
    otunredeemed 	0
    otoperator 	0
    otfollowers 	0
    gos 	0
    clanvanguards 	0
    ottrans 	0
    clangaia 	0
    otmed 	1234567890
    clansentinels 	0
    otarmedforces 	0
    playerid 	1234567890
    alienlevel 	0
    aliennextxp 	1500
    alienxp 	0
    insurancepercentage 	0
    prevmovementmode 	3
    teamside 	0
    absorbprojectileac 	0
    absorbmeleeac 	0
    absorbenergyac 	0
    absorbchemicalac 	0
    absorbradiationac 	0
    absorbcoldac 	0
    absorbnanoac 	0
    absorbfireac 	0
    absorbpoisonac 	0
    temporaryskillreduction 	0
    insurancetime 	0
    mapflags 	0
    changesidecount 	0
    specialattackshield 	0
    dmsmodifier 	0
    reflectreturnedmeleeac 	0
    shieldmeleeac 	0
------------------------------------

- 272.5, 531.4, 16.6 (272.5 531.4 y 16.6 1931)
- Pf Proxy: Model=51100:1931 GS=0 SG=0 R=1931, resource: 1931
- zone: 26, area: "Temple of Three Winds", room: "Future"
Server id: 639, character id: 50000:639, time: 2023-12-28 06:08:25 (UTC)
Version: 00.3.6_EP1, map id: NOT FOUND, build: 52606.

 ```

**Comment by wowpin** [comment]
Tried to replicate this just now, as I also noticed a double Uklesh spawn yesterday (killing one of them spawned Khalum and later Azzy, killing the other one did not). 

At the moment only one Uklesh is present and killing it does not spawn Khalum.

**Comment by RaggyLP** [comment]
Confirmed. There are 2 Uklesh currently present in the intial PF, one doesn't spawn Khalum, the other does.

![image](https://github.com/project-rubika/issues/assets/156821187/a643fc84-4c73-4069-ba84-681f5692b0a1)
![image](https://github.com/project-rubika/issues/assets/156821187/d5d7fe0e-935f-4d44-8ac0-a743f3d1e364)

**Comment by Starbouy138** [comment]
Just killed one of the two Uklesh and it did not spawn Khalum, the second Uklesh is unattackable and ignores my prescence.

**Comment by Starbouy138** [comment]
So today I logged into the game and my toon was already in TOTW, I killed Exarchs and Uklesh. I logged out inside TOTW as soon as Khalum popped and I had to leave the house. I logged back in about 30 minutes later to find that Khalum had despawned and two Uklesh were in position, one unattackable and facing backwards and the other died without spawning Khalum. I wonder if that could be a cause for the dual Uklesh bug.
To further this, I TP to my same location in TOTW and see the one exarch I had just killed still gone and no Uklesh, meaning I had killed one and the unattackable one was now gone as well. I decided to wait for Uklesh to spawn so I could kill him to see if Khalum would spawn, and about 5 minutes later while waiting, the client crashed at 4:44 pm CST. That may be unrelated.
